### PR TITLE
Fetch the languages from the servlet and display them on the DOM.

### PR DESCRIPTION
**Pull request by amuamushu**
This fetches the list of languages as the user inputs them.

No Selector if user hasn't inputed a url: https://screenshot.googleplex.com/69XQ6NaDe4LrXHE.png
Select if user inputs a valid url: https://screenshot.googleplex.com/5sSJzijXJ6UewLN.png
No Selector if user inputs an invalid url: https://screenshot.googleplex.com/33suqhNR8dNeZgv.png
Message instead of a selector if there is no transcripts available: https://screenshot.googleplex.com/7gmgtbZQv7CQNDW.png

This is the 3rd of 4 pull requests and will only be merged in if the others are approved in time.

1st PR: #362
2nd PR: #363
3rd PR: (This one) #372
4th PR: #378

### CODE REVIEW

**Review by f-camp — APPROVED** [review]

File: src/main/webapp/lectures/index.html [hunk 1]
```diff
@@ -58,7 +58,16 @@ <h3 class="text-center">Add a Lecture</h3>
...
-            <input type="url" class="form-control" placeholder="Lecture YouTube URL" name="link-input" required />
+            <input
+              type="url"
+              class="form-control"
+              placeholder="Lecture YouTube URL"
+              name="link-input"
+              oninput="fetchAndDisplayTranscriptLanguages(this)"
+              required
+            />
+            <br />
```
Comment (billie-cat) [inline src/main/webapp/lectures/index.html:69]: nit: I don't think there should be an empty space here.

Comment (billie-cat) [reply src/main/webapp/lectures/index.html:69]: I noticed this is everywhere in the code. Is this normal?

File: src/main/webapp/lectures/index.html [hunk 2]
```diff
@@ -58,7 +58,16 @@ <h3 class="text-center">Add a Lecture</h3>
...
             <br />
-            <input type="url" class="form-control" placeholder="Lecture YouTube URL" name="link-input" required />
+            <input
+              type="url"
+              class="form-control"
+              placeholder="Lecture YouTube URL"
+              name="link-input"
+              oninput="fetchAndDisplayTranscriptLanguages(this)"
+              required
+            />
```
Comment (billie-cat) [inline src/main/webapp/lectures/index.html:68]: Nit: I would move this to the previous line.

File: src/main/webapp/lectures/lectures.js [hunk 3]
```diff
@@ -52,3 +57,62 @@ function createLectureListItem(lecture) {
 
   return lectureLink;
 }
+
+/**
+ * Fetches and displays the list of transcript languages.
```
Comment (billie-cat) [inline src/main/webapp/lectures/lectures.js:62]:
```suggestion
 * Fetches and displays the list of available transcript languages.
```

**Review by billie-cat — CHANGES_REQUESTED** [review]

File: src/main/webapp/lectures/index.html (hunk 2 above)
Comment (amuamushu) [reply src/main/webapp/lectures/index.html:68]: I think the linter does this automatically, should I still fix this?

File: src/main/webapp/lectures/index.html (hunk 1 above)
Comment (amuamushu) [reply src/main/webapp/lectures/index.html:69]: I believe so, the linter made this change.

File: src/main/webapp/lectures/lectures.js (hunk 3 above)
Comment (amuamushu) [reply src/main/webapp/lectures/lectures.js]: added, thanks!

File: src/main/webapp/lectures/index.html (hunk 2 above)
Comment (billie-cat) [reply src/main/webapp/lectures/index.html:68]: If the linter does this, then keep it as is.

File: src/main/webapp/lectures/index.html (hunk 1 above)
Comment (billie-cat) [reply src/main/webapp/lectures/index.html:69]: Ah ok, thanks

**Review by billie-cat — APPROVED** [review]

**Pull request merged**